22.task.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	<script>
+		function nameFirst(init=0) {
+			let x = init;
+			return function(y) {
+				x += y
+				console.log(x)
```

Comment:
Use `console.log` only for development, debugging

Don't use it for production, your function must return a value

Use `console.log` after the function call and log the returned value

```
console.log(sum(3));
```